<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>953</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="lineEditName">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>40</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditSurname">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditAge">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>40</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditClass">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>40</y>
      <width>113</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Surname:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Age:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Class:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>0</y>
      <width>141</width>
      <height>21</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Student</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>prowadzacy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>katedra</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButtonInsert">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>40</y>
      <width>131</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Insert</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidgetAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>900</width>
      <height>431</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>953</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
